.gitignore

```diff
@@ -15,4 +15,5 @@ venv/
 
 .coverage
 .coverage.*
+etherscan_cache.sqlite
```

Comment:
should remove this once we move to the global `~/.cache/titanoboa` cache directory.

also, i wonder if it is thread-safe. like if there are multiple boa instances running, do they error out connecting to the sqlite database?

Reply:
> also, i wonder if it is thread-safe. like if there are multiple boa instances running, do they error out connecting to the sqlite database?

hm i guess it's no problem - https://stackoverflow.com/a/10387821